cli/user/src/test/java/io/pravega/cli/user/AbstractUserCommandTest.java

```diff
@@ -19,29 +20,33 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 
+import static io.pravega.cli.user.TestUtils.createPravegaCluster;
+import static io.pravega.cli.user.TestUtils.setInteractiveConfig;
+
 public abstract class AbstractUserCommandTest {
 
-    // Setup utility.
-    protected static final SetupUtils SETUP_UTILS = new SetupUtils();
+    protected static final AtomicReference<ClusterWrapper> CLUSTER = new AtomicReference<>();
```

Comment:
Why are you using `AtomicReference` for `CLUSTER` and `CONFIG`?

Since this `CLUSTER` variable is static, it will be shared across all classes that use it. This will cause a synchronization issue across different classes that use this or call `setUpCluster()`. You should instantiate this in the respective classes instead so that you don't prevent tests from different classes from running parallelly. Or, alternatively, you can have the `setUpCluster()` return a new instance.

Reply:
As discussed, avoiding the use of `AtomicReference`.